Formula/wxwidgets.rb

```diff
@@ -62,7 +70,7 @@ def install
       args << "--with-libiconv"
     end
 
-    system "./configure", *args
+    system "./configure", *std_configure_args, *args
```

Comment:
I'm guessing `--disable-debug` is causing dependents to fail. Previously we were using default:
```
DEFAULT_wxUSE_DEBUG_FLAG=yes
DEFAULT_wxUSE_DEBUG_INFO=no
```
Strictly disabling sets:
```
DEFAULT_wxUSE_DEBUG_FLAG=no
DEFAULT_wxUSE_DEBUG_INFO=no
```
Which causes `-DwxDEBUG_LEVEL=0`. Not sure what impact this has, so may just switch back to previous args